WcaOnRails/lib/advancement_condition.rb

```diff
@@ -83,12 +83,13 @@ def self.wcif_type
   end
 
   def to_s(round, short: false)
+    round_format = I18n.t("formats#{".short" if short}.#{round.format_id}")
     if round.event.timed_event?
-      I18n.t("advancement_condition#{".short" if short}.attempt_result.time", time: SolveTime.centiseconds_to_clock_format(attempt_result))
+      round_format + I18n.t("advancement_condition#{".short" if short}.attempt_result.time", time: SolveTime.centiseconds_to_clock_format(attempt_result))
```

Comment:
For our locals, you can have multiple inputs in inject into the string. The strings in en.yml should be fully readable on their own. So for example, The en.ymal variable can look like:
`time: "%{round_format} < %{time}"`
and then the response in the to_s method can look like:
`I18n.t("advancement_condition#{".short" if short}.attempt_result.time", round_format: round_format, time: SolveTime.centiseconds_to_clock_format(attempt_result))`

Reply:
Thanks for pointing this out! I didn't know it could be used that way.